docs/content/how-to/openstack/create-openstack-cluster.md

```diff
@@ -115,42 +128,45 @@ export FLAVOR="m1.large"
 
 # Optional flags:
 # External network to use for the API and Ingress endpoints.
```

Comment:
Doesn't API run in the management cluster? The external network just for the Ingress endpoint as I understand it.